src/DssSpell.sol

```diff
@@ -112,6 +125,17 @@ contract DssSpellAction is DssAction {
             ARB_GAS_PRICE_BID,
             ARB_MAX_SUBMISSION_COST
         );
+
+        // ------------------ Pause Starknet Goerli L2DaiTeleportGateway -----------------
+        // Forum: https://forum.makerdao.com/t/community-notice-pecu-to-redeploy-teleport-l2-gateways/19550
+        // L2 Spell to execute via STARKNET_GOV_RELAY:
+        // src: https://github.com/makerdao/starknet-spells-goerli/blob/b7ca995cf1d266aa2382d85e35a86b4fae52aa15/src/spell.cairo
+        // contract: https://testnet.starkscan.co/class/0x00a052591661d7e249b46a1084c63b14dae6aa8b1a56ab3f7df8c8add1c374b1#overview
+        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay{value: STA_L1_CALL_VALUE}(STARKNET_L2_SPELL);
+
+        // disallow legacy bridge on escrow
+        StarknetEscrowLike(STARKNET_ESCROW).approve(DAI, STARKNET_DAI_BRIDGE_LEGACY, 0);
```

Comment:
Can you please give some context to why we are doing this now.

Reply:
It is part of the two step bridge upgrade process described here: https://github.com/makerdao/starknet-dai-bridge#upgrades. It was not executed earlier as there were a lot of other stuff going into the spells at the end of last year. For the same reasons L2 part of the spell includes also code the cleans up GovRelay upgrade that happend in October '22.